tool/Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM nextcloud:27.0.0
+FROM nextcloud:27.0.0@sha256:a4181a159a0eb4d33cf7c5a05a24ca39c1ff6b7df29a125b5e47ab2ea2863bd1
```

Comment:
I don't like this, @Leptopoda what do you think?

Reply:
The explanation at https://docs.renovatebot.com/docker/#digest-pinning makes sense though :thinking:

Reply:
I don't see any benefit nor downside.

Reply:
IMO it looks a bit uglier and hard to update manually, but since we now have proper automatic updates nobody should touch this anymore anyway